cli/src/main.rs

```diff
@@ -91,6 +91,24 @@ fn run() -> Result<(), CliError> {
         )
     );
 
+    #[cfg(feature = "keygen")]
+    {
+        use clap::{Arg, SubCommand};
+
+        app = app.subcommand(
+            SubCommand::with_name("keygen")
+            .about("Generates secp256k1 keys to use when signing circuit proposals in user home directory")
```

Comment:
May be used for signing other things in the future, so the 'signing circuit proposals' is likely to become outdated.